tailcfg/c2ntypes.go

```diff
@@ -50,5 +50,5 @@ type C2NUpdateResponse struct {
 	Supported bool
 
 	// Started indicates whether the update has started.
-	Started bool
+	Started bool `json:",omitempty"`
```

Comment:
Do we really want this to be omitted? It seems like `false` is an interesting value, not just an absence. (I realize Go doesn't care, but as a matter of protocol it seems useful to have explicit).